exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/MathFunctionsVarcharUtils.java

```diff
@@ -18,15 +18,24 @@
 
 package org.apache.drill.exec.expr.fn.impl;
 
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-//import static java.lang.Float.NaN;
-
 public class MathFunctionsVarcharUtils {
 
-  public String validateInput(String input) {
+  public static boolean isValid(String input) {
+    if (StringUtils.isEmpty(input)) {
+      return false;
+    } else {
+      input = input.trim();
+      return NumberUtils.isCreatable(input);
+    }
+  }
 
+  public static String validateInput(String input) {
```

Comment:
@cgivre, thanks for adding the `NumberUtils.isCreatable` function to check for a valid number. Should I get rid of the `validateInput()` method?

Reply:
Yeah, I don't think it's necessary.